### PR TITLE
feat: cleanup code and improvements

### PR DESCRIPTION
**Pull request by Anish-Shobith**
Signed-off-by: Anish-Shobith <anish.shobith19@gmail.com>

## Details
Name: Cleanup code
Code Review: @XynoxTheDev  @officialpiyush 
- [x] Cleanups of code
- [x] Prettier
- [x] Eslint
- [x] Improvements in some parts of code
- [x] Better Output messages at some place.

## Checklist

- [x] Have you followed the [contributing guidelines](https://github.com/Dude-Perfect-Discord-Bot/Dude-Perfect/blob/main/.github/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

> **Note**: We will close your PR without comment if you do not check the boxes above and provide ALL requested information.

### CODE REVIEW

**Review by officialpiyush — APPROVED** [review]
LGTM

**Comment by Anish-Shobith** [comment]
Don't Merge if you finished review too!

File: src/commands/Fun/8ball.js [hunk 1]
```diff
@@ -1,49 +1,61 @@
...
+            start:
+              '<a:RedTick:760514410115498025> **You need to ask a question so that I can answer to it!**',
+          },
+        },
+      ],
+    });
+  }
+
+  exec(message) {
+    const ball = [
```
Comment (XynoxTheDev) [inline src/commands/Fun/8ball.js:29]: Can we use a certain API for these replies?

Comment (Anish-Shobith) [reply src/commands/Fun/8ball.js:29]: We could but it can avoid fetching from the api and it saves time , and who knows when it will die (API)

**Review by XynoxTheDev — APPROVED** [review]
:+1:

**Review by XynoxTheDev — APPROVED** [review]
:+1:

**Review by XynoxTheDev — CHANGES_REQUESTED** [review]
Hide the `Owner` category in the help command and only the owner of the bot can view that command category. In this way, it won't confuse users between `Bot Owner` & `Guild Owner`.

**Comment by Anish-Shobith** [comment]
The commit f4c027c makes changes `help` command. It is now able to display per category, per command and everything, filters Owner only commands if the author is not a owner.

**Comment by Anish-Shobith** [comment]
> Hide the `Owner` category in the help command and only the owner of the bot can view that command category. In this way, it won't confuse users between `Bot Owner` & `Guild Owner`.
f4c027c

**Review by XynoxTheDev — APPROVED** [review]
LGTM :+1:

**Pull request merged**